### ISSUE TITLE
[Bug] Player doesn't get a hint to build a GP when only building a reception

### ISSUE THREAD

**Issue by Alberth289346** [CorsixTH/CorsixTH#1877]
#### Describe the issue

Player doesn't get a hint to build a GP and/or a fax to do that.

#### Steps to Reproduce
1. Start new game (picked Example Town).
2. Build a reception desk. (You get a hint from the advisor to hire a receptionist.)
3. Hire a receptionist.
4. Open the hospital (click the timer away).
5. Wait for new patient visiting the reception.
6. Patient walks away (often at least) from the reception and gets a question mark above its head.
7. Nothing else happens, aside from new patients arriving.

#### Expected Behavior

A hint or a fax to build a GP.

#### System Information
**CorsixTH Version:** 6168f3186913b2451df5c2427873f58e3ff949e2

#### Gamelog.txt
Nothing useful:
``` text
Created game with savegame version 155.
```

### Additional Info
![Screenshot from 2021-05-15 14-02-48](https://user-images.githubusercontent.com/3254232/118360291-39ced200-b587-11eb-83e7-9e4a68fcf7ca.png)

Note the lack of fax icons at the bar.

**Comment by lewri** [comment]
Confirmed present in 0.64 also.

Looking into this on the original -- the game actually prevents the patient using the Reception until you have a GP Office.
As such, we should probably mimic that behaviour. Patients still queue at Reception though.

**Comment by lewri** [comment]
Also to add -- I agree a hint is also needed in this odd case. Something the adviser would say e.g.

``Patients have arrived at your hospital but you don't have a GP Office for them to go to``

**Comment by mugmuggy** [comment]
Yep original they wander around essentially waiting for a GPs office to be built. They don't actually progress past reception, which they do in Corsix.

I guess the idea is you play with original campaign with the tutorial and then you know how to do it from them on.
A message or something is essentially a todo in seek_room.
https://github.com/CorsixTH/CorsixTH/blob/2e82bf2e7f6b0fa19990eb80623751392ada75b8/CorsixTH/Lua/humanoid_actions/seek_room.lua#L390-L397
